packages/clerk-js/src/ui/components/OrganizationSwitcher/OtherOrganizationActions.tsx

```diff
@@ -86,7 +116,139 @@ export const OrganizationActionList = (props: OrganizationActionListProps) => {
           </PreviewButton>
         ))}
       </Box>
+
+      {(userInvitations.count ?? 0) > 0 && (
+        <Flex
+          direction={'col'}
+          elementDescriptor={descriptors.organizationSwitcherPopoverInvitationActions}
+        >
+          <Text
+            variant={'smallRegular'}
+            sx={[
+              t => ({
+                minHeight: 'unset',
+                height: t.space.$12,
+                padding: `${t.space.$3} ${t.space.$6}`,
+                display: 'flex',
+                alignItems: 'center',
+              }),
+            ]}
+            // Handle plurals
+            localizationKey={localizationKeys(
+              (userInvitations.count ?? 0) > 1
+                ? 'organizationSwitcher.invitationCountLabel_many'
+                : 'organizationSwitcher.invitationCountLabel_single',
+              {
+                count: userInvitations.count,
+              },
+            )}
+          />
+          <Box
+            sx={t => ({
+              maxHeight: `calc(4 * ${t.sizes.$12})`,
+              overflowY: 'auto',
+              ...common.unstyledScrollbar(t),
+            })}
+          >
+            {userInvitations?.data?.map(inv => {
+              return (
+                <InvitationPreview
+                  key={inv.id}
+                  {...inv}
+                />
+              );
+            })}
+
+            {(userInvitations.hasNextPage || userInvitations.isFetching) && (
+              <Box
+                ref={ref}
+                sx={[
+                  t => ({
+                    width: '100%',
+                    height: t.space.$12,
+                    position: 'relative',
+                  }),
+                ]}
```

Comment:
(question) why is this `sx` prop an array while others are functions?

Reply:
Tbh I don't know why, but i know that it is supported and widely used across the code base

Reply:
But i'm updating these to be function in order for the sx props to be consistent inside this file